.github/CODEOWNERS

```diff
@@ -84,9 +84,6 @@ Makefile                                                 @getsentry/owners-sentr
 
 ## GitHub Routing Automations - notion.so/473791bae5bf43399d46093050b77bf0
 /.github/labels.yml                                        @getsentry/dev-infra
-/.github/workflows/react-to-product-owners-yml-changes.yml @getsentry/dev-infra
```

Comment:
Are we deleting https://github.com/getsentry/security-as-code/blob/f53fa2b3f98f786f6247c281d345cce4c87a591c/rbac/lib/product-owners.yml? These automations help keep sentry and security as code synced so product areas stay consistent, and we have clear ownership of product areas defined

Reply:
Yep, those are used by eng-pipes for sending out slack notifications for triaging github issues:
https://github.com/getsentry/eng-pipes/blob/c61130e29add77063054c98eeafd767318937826/product-owners.yml